.ci-operator.yaml

```
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: rhel-8-release-golang-1.15-openshift-4.8
+  tag: rhel-8-release-golang-1.16-openshift-4.8
```

See https://github.com/openshift/cluster-etcd-operator/pull/607

We need 
```suggestion
  tag: golang-1.16
```

---

actually this one might work. ckao uses the same.

---

I left it as is to match with ckao and pushed the change with go.sum.